services/callback/src/main/java/app/coronawarn/server/services/callback/controller/CallbackController.java

```diff
@@ -44,7 +46,8 @@ public CallbackController(FederationBatchInfoService federationBatchInfoService)
   public ResponseEntity<Void> handleCallback(@RequestParam String batchTag,
       @NotNull @DateTimeFormat(iso = ISO.DATE) @RequestParam LocalDate date) {
     logger.info("BatchInfo with tag {} and date {} received from federation gateway.", batchTag, date);
-    FederationBatchInfo federationBatchInfo = new FederationBatchInfo(batchTag, date);
+    // TODO check how to integrate swiss?
```

Comment:
Would this TODO be developed in another PR ?

Reply:
yes @EugenM-SAP